x-pack/plugins/security_solution/public/overview/components/detection_response/index.ts

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
We decided to not include this file, components should be imported from its own component folder

Reply:
removed 🍻